project-icp/bee-pollinator-app
# Feature/touchup map

This adds basemaps to the UI and updates the leaflet controls to reflect the new design. I used placeholder icons for the controls, and I deferred changing stylesheets in favor of #20 , which might require rebasing of one of the branches.

To test:
* clone this branch and bring up the environement
* navigate to `http://localhost:8000`
* check that the leaflet map has a basemap
* check that the controls resemble the design here: https://github.com/project-icp/bee-pollinator-app/files/576074/2016_10_27_bee-pollinator_v1.pdf
* click on the basemap toggle control a few times (the second button in the lower left) and check that it switches the map
* do some panning and zooming and toggle at random
* check that that the existing controls behave as expected; e.g. the search control searches, the zoom buttons zoom
* check the browser console for any errors

Connects #21 Connects #26
